platforms/intel-iot/edison/sparkfun_pinmap.go

```diff
@@ -0,0 +1,116 @@
+package edison
+
+var sparkfunPinMap = map[string]sysfsPin{
+  "12": sysfsPin{
```

Comment:
Shouldn't this be labeled "GP12" to exactly match the labels on the board? Likewise with the reset of the pins.